migrations/6_create_MVP_Tables.up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE users (
+    PRIMARY KEY (user_id),
+    user_id         VARCHAR(255)            NOT NULL,
+    age             INT UNSIGNED,
+    calorie_goal    INT UNSIGNED,
+    display_name    VARCHAR(255),
+    height_inches   INT UNSIGNED,
+    sex             ENUM('Female', 'Male', 'Other'),
+    weight_pounds   INT UNSIGNED
+);
+
+CREATE TABLE exercise_records (
```

Comment:
Looks like you've copied migration 5 with a higher number. On deploy, the service is trying to apply this migration on top of migration 5 (the currently deployed version), and failing. Table creation fails because these already exist.

Migrations are all applied in order. 6 does not override 5, but rather "migrates" from the state resulting from 1-5 to a new state. That's why we also define the "down" migrations so that we can attempt to roll a change back to the previous state if needed.

Rather than redefining everything, we should alter the existing tables to add additional enum values for the required columns. Check this out as an example: https://stackoverflow.com/questions/1501958/how-do-i-add-more-members-to-my-enum-type-column-in-mysql

Reply:
Ah I see, so we run through all the up migrations in order. Thanks!